src/main.js

```diff
@@ -100,7 +100,7 @@ function addToolbarButton() {
     // the toolbar item to any new toolbars.
     observeEl(toolbar, () => {
       let toolbarGroup = select.all(
-        '.toolbar-commenting > :not([class*="--hidden"]):not(button)',
+        '.ActionBar-item-container > :not([class*="--hidden"]):not(button)',
```

Comment:
This can be simplified a bit:
```suggestion
      const toolbarGroup = select(
        '.ActionBar-item-container',
        toolbar
      )
```